tests/EnvironmentTest.php

```diff
@@ -89,8 +89,8 @@ public function escapingStrategyCallback($name)
     public function testGlobals()
     {
         // to be removed in 2.0
-        $loader = $this->getMockBuilder('\Twig\Tests\EnvironmentTestLoaderInterface')->getMock();
-        //$loader = $this->getMockBuilder(['\Twig\Loader\LoaderInterface', '\Twig\Loader\SourceContextLoaderInterface'])->getMock();
+        $loader = $this->createMock('\Twig\Tests\EnvironmentTestLoaderInterface');
+        //$loader = $this->createMock(['\Twig\Loader\LoaderInterface', '\Twig\Loader\SourceContextLoaderInterface']);
```

Comment:
any particular reason to keep a commented code? same bellow

Reply:
It was to remember the code to write in 2.x,  not really needed anymore indeed.